boards.txt

```diff
@@ -36447,3 +36447,205 @@ weact_studio_esp32c3.menu.EraseFlash.all=Enabled
 weact_studio_esp32c3.menu.EraseFlash.all.upload.erase_cmd=-e
 
 ##############################################################
+
+aslcanx2.name=AutosportLabs ESP-CAN-X2
+aslcanx2.vid.0=0x303a
+aslcanx2.pid.0=0x1001
+
+aslcanx2.bootloader.tool=esptool_py
+aslcanx2.bootloader.tool.default=esptool_py
+
+aslcanx2.upload.tool=esptool_py
+aslcanx2.upload.tool.default=esptool_py
+aslcanx2.upload.tool.network=esp_ota
+
+aslcanx2.upload.maximum_size=1310720
+aslcanx2.upload.maximum_data_size=327680
+aslcanx2.upload.flags=
+aslcanx2.upload.extra_flags=
+aslcanx2.upload.use_1200bps_touch=false
+aslcanx2.upload.wait_for_upload_port=false
+
+aslcanx2.serial.disableDTR=false
+aslcanx2.serial.disableRTS=false
+
+aslcanx2.build.tarch=xtensa
+aslcanx2.build.bootloader_addr=0x0
+aslcanx2.build.target=esp32s3
+aslcanx2.build.mcu=esp32s3
+aslcanx2.build.core=esp32
+aslcanx2.build.variant=aslcanx2
+aslcanx2.build.board=ESP32S3_DEV
```

Comment:
I recommend to change the build.board to be your board specific name, for example ASL_CAN_X2.

Reply:
Thanks, I changed it to ASL-CAN-X2.

Reply:
Please use underscores instead of dashes. Thanks!

Reply:
Done! :) thank you.